packages/css/src/plugins/sort-shorthand-declarations.ts

```diff
@@ -9,15 +9,17 @@ const findDeclaration = (node: ChildNode): Declaration | undefined => {
   }
 
   if ('nodes' in node) {
-    if (node.nodes.length === 1 && nodeIsDeclaration(node.nodes[0])) {
-      return node.nodes[0];
-    }
+    // Return the first node that is a declaration, if we find one
```

Comment:
This may not be correct at all, eg. if Compiled ever tries to sort this, it's kind of uncharted territory…`global` will come second here 🥴
```tsx
.global {padding-left: 8px;all: unset}
._abcd1234{padding: 8px}
```

Reply:
I think we could technically do a deep recursive sort _somehow_, eg. compare the scores of all children vs. another child, but I'm expecting in real-world Compiled, the above is not a legitimate scenario and instead using the first property's "score" is fine.

Reply:
I think if I learnt anything from this, it's that it's safer to ask devs not to mix shorthand and longhand properties at all, than to try to work with it 😅